Dockerfile.QA

```diff
@@ -165,51 +165,51 @@ RUN mkdir -p qa/custom_models/custom_sequence_int32/1 && \
         qa/L0_perf_nomodel/custom_models/custom_zero_1_float32/1/. && \
     mkdir -p qa/L0_perf_pyclients/custom_models/custom_zero_1_int32/1 && \
     cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_perf_pyclients/custom_models/custom_zero_1_int32/1/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_perf_analyzer/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_large_payload/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_model_config/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_model_queue/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_simple_lib/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_pinned_memory/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_io/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_warmup/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_cmdline_trace/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_batcher/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_client_timeout/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_client_valgrind/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_client_memory_growth/. && \
-    cp /tmp/tritonbuild/tritonserver/build/test-util/install/lib/libidentity.so \
+    cp /tmp/tritonbuild/install/backends/identity/libtriton_identity.so \
         qa/L0_custom_legacy/. && \
```

Comment:
You can probably drop this test, its name suggests that it is not needed after we deprecate legacy custom backend

Reply:
Sure thin